Library/Formula/gvp.rb

```diff
@@ -17,11 +15,20 @@ def install
     system "make", "install"
   end
 
+  def caveats; <<-EOS.undent
+    To have gvp modify your environment, add the following at the end of your
+    .bashrc or .zshrc:
+
+      source gvp
```

Comment:
`gvp` is in the `PATH` since it’s in `#{prefix}/bin`, so users don’t need to use the full path. Thanks for the look at its manual, I’ll change the caveats.